pkg/liqo-controller-manager/namespace-controller/manage_namespace_labels.go

```diff
@@ -10,12 +12,35 @@ import (
 	"strings"
 )
 
+// This function sets remote Namespace name to default value "localName-clusterID" , and adds an Annotation to notify user
+func (r *NamespaceReconciler) checkMappingLabelDefaulting(n *corev1.Namespace) error {
+	clustedID, err := liqocontrollerutils.GetClusterID(r.Client)
+	if err != nil {
+		return err
+	}
+
+	n.Labels[mappingLabel] = fmt.Sprintf("%s-%s", n.Name, clustedID)
+
+	if n.GetAnnotations() == nil {
+		n.Annotations = map[string]string{}
+	}
+	n.GetAnnotations()[mappingAnnotationDefaulting] = fmt.Sprintf("You have not specified a name for your remote Namespace, this is your default name: [%s]. Please read our documentation for more info [%s]",
+		n.GetLabels()[mappingLabel], liqocontrollerutils.DocumentationUrl)
```

Comment:
I feel referring to the home-page of the documentation is mostly useless, as the user would not know where to look for the information about this specific issue. Additionally, ATM there is no documentation about this feature, hence it is even more misleading.

Reply:
Yes you are right, thank you